### PR TITLE
update react demo

### PR DESCRIPTION
**Pull request by mmacxm**

### CODE REVIEW

File: server/node/src/api.js [hunk 1]
```diff
@@ -3,12 +3,13 @@ import StatusCodes from 'http-status-codes'
...
 import {tickets as ticketsDb, users as usersDb} from './db.js'
 
+// all routes in this router is prefixed with /api, see ./server.js:42
 export const router = Router()
 const {OK, FORBIDDEN} = StatusCodes
 
 // resolve ticket
 router.post('/tickets/:id/resolve', async (req, res) => {
-  const ticket = ticketsDb[req.auth.tenant][req.params.id]
+  const ticket = new Map(Object.entries(ticketsDb[req.auth.tenant])).get(req.params.id)
```
Comment (mmacxm) [inline server/node/src/api.js:12]: fixes security issue: https://github.com/StyraInc/fetchdb/security/code-scanning/160

Comment (mmacxm) [reply server/node/src/api.js]: reverted

Comment (johanfylling) [reply server/node/src/api.js]: If we ended up changing this in the tutorial, we should keep this in sync: https://github.com/StyraInc/fetchdb/pull/10592#discussion_r1013440939

**Review by johanfylling — APPROVED** [review]
👍

**Pull request merged**